libs/blocks/marquee/marquee.js

```diff
@@ -114,7 +114,7 @@ export default async function init(el) {
     }
 
     let mediaCreditInner;
-    const txtContent = media?.lastChild.textContent.trim();
+    const txtContent = media?.lastChild?.textContent?.trim();
```

Comment:
This may have already been included in https://github.com/adobecom/milo/pull/1865

Reply:
Thanks! Yes oddly we both found this issue at the same time